File: DBBasic/output_obj/info_parcs_obj.cs
using DBBasic.Model;
using System;
using System.Collections.Generic;
using System.Text;

namespace DBBasic.output_obj
{
    public class info_parcs_obj : Serializable
    {
        public List<info_parc> parcs;
        public info_parcs_obj(List<info_parc> parcs)
        {
            this.parcs = parcs;
        }
    }
}

File: Socis/app/src/main/java/com/app/socis/input_obj/info_parcs_obj.java
package com.app.socis.input_obj;

import com.app.socis.Serializable;
import com.app.socis.model.parc;

import java.util.List;

public class info_parcs_obj extends Serializable {
	public List<parc> parcs;
	public info_parcs_obj(List<parc> parcs)
	{
		this.parcs = parcs;
	}
}
